README.md

```diff
@@ -83,6 +83,16 @@ Short vocabulary
 Download and build
 ==================
 
+Quick start with VS Code and Remote Container
+---------------------------------------------
+
+1. Download and install [Visual Studio Code](https://code.visualstudio.com/).
+2. Install the extension __Remote Development__.
+3. Clone _dlite_ and initialize git modules: `git submodule update --init`.
+4. Open the _dlite_ folder with VS Code.
+5. When presented with a notification, select __Reopen in container__. This will build the container and restart VS Code in it. This may take some time the first time as the Docker image must be built.
```

Comment:
Should there be a point about installing Docker?

I do not get a notification after opening _dlite_ in VS Code. It might be because that I am using [Fedora 31, which enables cgroup v2](https://www.redhat.com/sysadmin/fedora-31-control-group-v2) by default. To use Docker, it seems that I have to [revert back to cgroup v1](https://linuxconfig.org/how-to-install-docker-on-fedora-31).

Reply:
Ok, I might revise this description a bit. You should not need the notificaiton to open it.

Reply:
Also, I will take a look at the false positives. It is best if we can avoid adding suppressions, but getting checking in has value even with supression of some issues.

Reply:
I have reverted to cgroup v1 and can now run docker again! I still don't get any __Reopen in container__ option in VS Code, but the docker tab shows a list of containers and images. Is there a menu option?